Tests/Common/Module/ModuleTopLevelObjectTest.swift

```diff
@@ -37,7 +37,7 @@ class ModuleTopLevelObjectTest: UnitTest {
                 expectAllThreadsToComplete.fulfill()
             }
 
-            waitForExpectations(1) // test may take up to 1 second to finish because it is running so many times. CI server is a less powerful machine and this test is flaky when we set wait() for < 1 second.
+            waitForExpectations()
```

Comment:
Increasing the timeout originally was to make the test less flaky on CI. Since then, I have realized that this timeout does not reduce flakiness so, removing it.